lib/nautic_net/serial/six_fab/driver.ex

```diff
@@ -0,0 +1,30 @@
+defmodule NauticNet.Serial.SixFab.Driver do
+  @moduledoc """
+  For interacting with Sixfab GPS from their Raspberry Pi 4G/LTE Cellular Modem Kit.
+  """
+  @behaviour NauticNet.Serial.Driver
+  require Logger
+
+  @serial_write_name "ttyUSB2"
+  @serial_read_name "ttyUSB1"
```

Comment:
Could you be more explicit about these names? I think they should be more like
```suggestion
  @modem_serial_port_name "ttyUSB2"
  @gnss_serial_port_name "ttyUSB1"
```

Reply:
Sure. I was trying to follow the naming convention sixfab used in their python example.

```
portwrite = "/dev/ttyUSB2"
port = "/dev/ttyUSB1"
```